### PR TITLE
Fixes syndicate hardsuit remaining in EVA mode when head gear is disengaged.

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Makes it so the syndicate hardsuit will automatically disengage EVA mode when the helmet is toggled off. 

## Why It's Good For The Game

In the moment, it can be very hard to notice you are moving slowly as a nuke op when you disengage the helmet. Not a lot of people realise that the hardsuit is still slow while the helmet is off if the helmet was taken off during EVA mode.
This makes it so the suit is forced into combat mode when you lower the helmet.

## Changelog
:cl:
fix: Syndi hardsuit will now automatically engage combat mode when the helmet is lowered.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Wrong branch

**Comment by IllotAFV** [comment]
This is actually really good.

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -354,6 +350,45 @@
...
+	syndieHelmet.activate_combat_mode()
+	syndieHelmet.update_icon()
+	for(var/X in syndieHelmet.actions)
+		var/datum/action/A = X
+		A.UpdateButtonIcon()
+	//Update icon
+	icon_state = "hardsuit[syndieHelmet.on]-[syndieHelmet.item_color]"
+	update_icon()
+	//Update the suit to non-combat mode
+	activate_combat_mode()
```
Comment (park66665) [inline code/modules/clothing/spacesuits/hardsuit.dm]:
```suggestion
	//Update the suit to non-combat mode
	activate_combat_mode()
	//Update icon
	icon_state = "hardsuit[syndieHelmet.on]-[syndieHelmet.item_color]"
	update_icon()
```

`syndieHelmet.on` is changed at `activate_combat_mode()`. This causes a minor visual bug where the hardsuit looks likes EVA mode while being in combat mode (the differences are barely recognizable anyway)

**Review by park66665 — APPROVED** [review]
YOLO

**Pull request merged**